rust-src/id/src/account_holder.rs

```diff
@@ -496,29 +866,78 @@ pub fn commitment_to_share_and_rand<C: Curve>(
 
     let cmm = utils::commitment_to_share(&share_number.to_scalar::<C>(), coeff_commitments);
 
-    let cmm_share_randomness_scalar =
-        utils::evaluate_poly(coeff_randomness, &share_number.to_scalar::<C>());
-    let rnd = PedersenRandomness::new(cmm_share_randomness_scalar);
-    (cmm, rnd)
+    let cmm_share_randomness_scalar =
+        utils::evaluate_poly(coeff_randomness, &share_number.to_scalar::<C>());
+    let rnd = PedersenRandomness::new(cmm_share_randomness_scalar);
+    (cmm, rnd)
+}
+
+/// Generates a credential deployment info and outputs the randomness used in
+/// commitments. The randomness should be stored for later use, e.g. to open
+/// commitments later on. The information is meant to be valid in the context of
+/// a given identity provider, and global parameter.
+/// The 'cred_counter' is used to generate a new credential ID.
+pub fn create_credential_old<
```

Comment:
Will this be removed? Otherwise rename to `_v0` or something more meaningful than `_old`.

Reply:
To be removed.